packages/gestalt/src/Dropdown.js

```diff
@@ -88,7 +88,7 @@ type IdealDirection = 'up' | 'right' | 'down' | 'left';
 type Props = {|
   anchor?: ?HTMLElement,
   children: Node,
-  dangerouslyRemoveLayer?: boolean,
+  isWithinFixed?: boolean,
```

Comment:
Can we change this to `isWithinFixedContainer`? It's not clear what "fixed" we're referring to here

Reply:
+1

Reply:
Changed